test/ssltest_old.c

```diff
@@ -453,6 +474,42 @@ static int verify_alpn(SSL *client, SSL *server)
     return -1;
 }
 
+#ifndef OPENSSL_NO_TLSEXT
+
+static int cb_ticket0(SSL* s, unsigned char* key_name, unsigned char *iv, EVP_CIPHER_CTX *ctx, HMAC_CTX *hctx, int enc)
+{
+    return 0;
+}
+
+static int cb_ticket1(SSL* s, unsigned char* key_name, unsigned char *iv, EVP_CIPHER_CTX *ctx, HMAC_CTX *hctx, int enc)
+{
+    static unsigned char key[] = { 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16 };
+    static char name[] = "ticket11ticket11";
+    if (SSL_get_options(s) & SSL_OP_NO_TICKET)
+        return 0;
+    if (enc) {
```

Comment:
Afaict, there is no reason to do real encryption in the test? You should treat the test callback as an application "black box", i.e., it should be as simple as possible, only do the minimum needed for the test, and do that obviously correctly. See e.g. https://github.com/openssl/openssl/blob/master/test/handshake_helper.c#L43

Also please use descriptive names instead of "cb_ticket1", "cb_ticket2" etc.